code/modules/mob/living/simple_animal/hostile/mining_mobs.dm

```diff
@@ -329,7 +329,7 @@
 	mouse_opacity = 2
 	move_to_delay = 40
 	ranged = 1
-	ranged_cooldown_time = 80
+	ranged_cooldown_time = 140
```

Comment:
might be a bit too much, though I frankly have no clue what the correct value is anymore.

Goliaths right now are definitely scary, but they also encourage team work, so ??